### PR TITLE
Istio Version documentation

### PR DESCRIPTION
**Pull request by triffer**
**Description**

Changes proposed in this pull request:

- Automation to add Istio and Envoy version for new releases
- Add Istio and Envoy version to README
- Update issue template for Istio updates to include updating the versions in the README

**Related issues**

- https://github.com/kyma-project/istio/issues/610

### CODE REVIEW

**Review by kolodziejczak — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ Istio is an open-source service mesh that provides a uniform way to manage, conn
 
 Kyma Istio Operator is a component of the Kyma runtime that handles the management and configuration of the Istio service mesh. Within Kyma Istio Operator, [Istio Controller](/docs/user/00-10-overview-istio-controller.md) is responsible for installing, uninstalling, and upgrading Istio.
 
+**Istio version:** 1.21.2
```
Comment (nataliasitko) [inline README.md:14]:
```suggestion
The latest release includes the following versions of Istio and Envoy:
**Istio version:** 1.21.2  
```

Comment (nataliasitko) [reply README.md:14]: I'd also add this information to the readme in the /user folder
https://github.com/kyma-project/istio/blob/main/docs/user/README.md

File: .github/ISSUE_TEMPLATE/istio_update.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ ACs:
 - [ ] Istio module upgrades with zero downtime - https://github.com/kyma-project/istio/issues/429
 - [ ] Tests and documentation updated if needed.
 - [ ] Verify that sidecars are in sync with Control Plane.
+- [ ] Istio and Envoy Version updated in the `README.md`. The scripts `scripts/get_module_istio_version.sh` and `scripts/get_module_envoy_version.sh` can be used to extract the versions.
```
Comment (nataliasitko) [inline .github/ISSUE_TEMPLATE/istio_update.md:22]:
```suggestion
- [ ] Istio and Envoy Version updated in the [`README.md`](https://github.com/kyma-project/istio) and [`/docs/user/README.md`](https://github.com/kyma-project/istio) files. You can use the scripts `scripts/get_module_istio_version.sh` and `scripts/get_module_envoy_version.sh` to extract the versions.
```

File: README.md (hunk 1 above)
Comment (nataliasitko) [reply README.md:15]: the one in the /user folder is displayed on the website and the other one is the main repo's readme, so I guess it's better to have duplicated. I totally get your point, though :)

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**